docs/sdks/client-sdks/ios.md

```diff
@@ -44,90 +72,220 @@ func segmentAssignmentLogger(assignment: Assignment) {
+
+```swift
+try EppoClient.initializeOffline(
+    sdkKey: "SDK-KEY-FROM-DASHBOARD",
+    initialConfiguration: try Configuration(
+        flagsConfigurationJson: Data(#"{ "pre-loaded-JSON": "passed in here" }"#.utf8),
+        obfuscated: false
+    )
+);
+
+...
+
+Task {
+    try await EppoClient.shared().load();
+}
+```
+
+As modern iOS devices have substantial memory, applications are often kept in memory across sessions. This means that the flag configurations are not automatically reloaded on subsequent launches.
+
+It is recommended to use the `load()` method to fetch the latest flag configurations when the application is launched.
+
+
+## Assignment Logger Schema
+
+
+The SDK will invoke the `logAssignment` function with an `Assignment` object that contains the following fields:
+
+| Field                     | Description                                                                                                              | Example                             |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
+| `allocation` (string)     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
+| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17" |
+| `featureFlag` (string)    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
+
+:::note
+More details about logging and examples (with Segment, Rudderstack, mParticle, and Snowplow) can be found in the [event logging](/sdks/event-logging/) page.
+:::
+
+## Appendix
+
+### Publishing releases
+
+Swift Package Manager relies on semantic versioning without a prefix, such as `v`.
+
+When publishing a release of the Swift SDK, use a git tag such as `3.2.1`.
```

Comment:
This is a developer-only documentation. Users don't need to know this
```suggestion
```

Reply:
Hahah yeah sorry should have included "docs" in the title 😅